src/apiary.apib

```diff
@@ -1815,27 +1815,26 @@ level `conversions` array.
 
 ## Creative Common (object)
 + campaign_id: 10l176 - ID of the campaign
++ creative_type_id: 1 (number, required)
 + active (YesOrNo)
-+ creative_type_id: 1 (number)
-+ creative_type
 + custom_append_url_parameters
 + custom_prepend_url_parameters
 + tags
 
 ## Creative Base (object)
 + Include Creative Common
 + description: Demo Text
-+ destination
++ destination: http://www.performancehorizon.com
 + default_specific_creative_id: 300396
 + creative_tracking_link
 + creative_tracking_link_rotate
 + dynamic_tracking_link
 + html_tracking_link
-+ width: 400
-+ height: 360
++ width: 400 (number)
++ height: 360 (number)
 + Include Time Range
 + preview: https://demo.prf.hn/preview/creativeref:300442
-+ advertiser_reference
++ advertiser_reference: Advertiser Reference
```

Comment:
I would change the example to something that would be obvious to an advertiser use case, maybe "Promotion July 2016" or something along those lines to suggest that they would tag it up to cross reference with some marketing activity.